app/TestTCP/testclose.cpp

```diff
@@ -187,6 +188,8 @@ TEST_F(TestEnv_Any, TestClose_Connect_CloseLater)
 	std::unordered_map<std::string, std::string> accept_env;
 	std::unordered_map<std::string, std::string> connect_env;
 
+	printf("[RANDOM_SEED : %s]\n", getenv("RANDOM_SEED"));
```

Comment:
I think you should have at most 3 of these statements.
In 3 different kind of environment setups in `testenv.hpp`.